pages/workspace/tracking-plan/exporting.mdx

```diff
@@ -6,10 +6,18 @@ title: 'Exporting'
 
 import Link from '../../../components/Link';
 
-Go to **Settings** :gear: > **Export** to export your full or partial tracking plan in a CSV or JSON format. (In beta, request access via support@avo.sh).
+<Link passHref href="/workspace/tracking-plan/publishing">Publishing integrations</Link> can be used for exporting your tracking plan, either one of, or a recurring publishing export using auto-publishing.
 
-You can customize which events to export by using the events filter. Events can be filtered by Sources, Destinations and Tags.
+
+To export your tracking plan on a JSON Schema format:
+
+1. Head to Tracking Plan > **Publish** in the sidebar of your Avo workspace
+2. Create a Webhook integration
+3. In the "Payload preview" section click "Click to view payload preview". When the preview has loaded you can either copy or download the JSON file. You can find more details on the format of the JSON in the <Link passHref href="/workspace/tracking-plan/publishing#tracking-plan-model">Publishing docs</Link>.
+<img src={require('../../../images/workspace/webhook-export.png')} />
+
+You can customize which events to export by using the events filter in the webhook configuration. Events can be filtered by Sources, Destinations and Tags.
 
 ## What's next?
 
-If you find yourself regularly exporting, you might want to <Link passHref href="/workspace/tracking-plan/publishing"><a>configure Publishing or Auto-Publishing</a></Link> for your tracking plan to consistently stay up to date with a downstream tool such as your analytics platform, your production time validation solution, your SQL table management, etc.
\ No newline at end of file
+If you find yourself regularly exporting, you might want to <Link passHref href="/workspace/tracking-plan/publishing"><a>configure Auto-Publishing</a></Link> for your tracking plan to consistently stay up to date with a downstream tool such as your analytics platform, your production time validation solution, your SQL table management, etc.
```

Comment:
And here ✌️

Reply:
This one is good, right? Fixed the other 2